src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/DeleteCompleteFilesTask.java

```diff
@@ -88,65 +88,41 @@ private void processCompleteFiles(CloudBlobContainer container) {
         );
     }
 
-    @SuppressWarnings("PMD.DataflowAnomalyAnalysis")
     private boolean tryProcessCompleteEnvelope(CloudBlobContainer container, Envelope envelope) {
-        boolean deleted = false;
+        boolean deleted;
+
+        String loggingContext = "File name: " + envelope.getZipFileName() + ", Container: " + container.getName();
 
         try {
-            CloudBlockBlob cloudBlockBlob = container.getBlockBlobReference(envelope.getZipFileName());
+            log.info("Deleting file. " + loggingContext);
```

Comment:
can be log specific parameterised :wink:

Reply:
fixed